arrow-libs/core/arrow-core-data/src/main/kotlin/arrow/core/Ior.kt

```diff
@@ -527,7 +527,43 @@ sealed class Ior<out A, out B> {
    */
   @Deprecated("Deprecated, use `leftOrNull` instead", ReplaceWith("leftOrNull()"))
   fun toLeftOption(): Option<A> =
-    fold({ Option.just(it) }, { Option.empty() }, { a, _ -> Option.just(a) })
+    fold({
+      /**
+       * Lifts a pure [A] value to [Option]
+       *
+       * {: data-executable='true'}
+       *
+       * ```kotlin:ank
+       * import arrow.core.Option
+       * fun main(args: Array<String>) {
+       * //sampleStart
+       * val result: Option<Int> = Option.just(1)
+       * //sampleEnd
+       * println(result)
+       * }
+       * ```
+       *
+       */
+      Some(it)
+    }, { None }, { a, _ ->
+      /**
+       * Lifts a pure [A] value to [Option]
+       *
+       * {: data-executable='true'}
+       *
+       * ```kotlin:ank
+       * import arrow.core.Option
+       * fun main(args: Array<String>) {
+       * //sampleStart
+       * val result: Option<Int> = Option.just(1)
+       * //sampleEnd
+       * println(result)
+       * }
+       * ```
+       *
+       */
+      Some(a)
+    })
```

Comment:
Did the IDE put this here when you removed the function?

```suggestion
    fold({ Some(it) }, { None }, { a, _ -> Some(a) })
```

Reply:
I think IDEA did something weird here when inlining references :/